### PR TITLE
PLANET-3408 Add available fields custom meta box

### PR DESCRIPTION
**Pull request by kirdia**

### CODE REVIEW

File: classes/controller/class-enform-fields-list-table.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+		parent::__construct(
+			[
+				'ajax' => false,
+			]
+		);
+
+		$this->error = '';
+	}
+
+	public function prepare_items() {
```
Comment (angtheod) [inline classes/controller/class-enform-fields-list-table.php]: Missing Doc comments for this and almost all the rest Class properties and methods.

File: classes/controller/class-enform-fields-list-table.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+			case 'id':
+			case 'name':
+			case 'property':
+			case 'tag':
+				return $item[ $column_name ];
+		}
+	}
+
+	public function column_actions( $item ) {
+		return '<button disabled>Add</button>';
```
Comment (angtheod) [inline classes/controller/class-enform-fields-list-table.php]: please register this for translation

File: classes/controller/class-enform-fields-list-table.php [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	}
+
+	/**
+	 * Extra controls to be displayed between bulk actions and pagination
+	 *
+	 * @param string $which
+	 */
+	protected function extra_tablenav( $which ) {
+		if ( ! empty( $this->error ) && 'top' === $which ) {
+			echo $this->error;
```
Comment (angtheod) [inline classes/controller/class-enform-fields-list-table.php]: phpcs complains that this should be escaped

File: classes/controller/class-enform-fields-list-table.php [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+			'name'     => __( 'Name', 'planet4-engagingnetworks' ),
+			'tag'      => __( 'Tag', 'planet4-engagingnetworks' ),
+			'property' => __( 'Property', 'planet4-engagingnetworks' ),
+			'actions'  => __( 'Actions', 'planet4-engagingnetworks' ),
+		];
+
+		return $columns;
+	}
+
+	protected function get_bulk_actions() {
```
Comment (angtheod) [inline classes/controller/class-enform-fields-list-table.php]: why is this one overridden?

File: classes/controller/class-enform-fields-list-table.php [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+	protected function column_default( $item, $column_name ) {
+		switch ( $column_name ) {
+			case 'id':
+			case 'name':
+			case 'property':
+			case 'tag':
+				return $item[ $column_name ];
+		}
+	}
```
Comment (angtheod) [inline classes/controller/class-enform-fields-list-table.php]: missing return statement

File: classes/controller/class-enform-fields-list-table.php (hunk 1 above)
Comment (kirdia) [reply classes/controller/class-enform-fields-list-table.php]: Added comments :+1:

File: classes/controller/class-enform-fields-list-table.php (hunk 4 above)
Comment (kirdia) [reply classes/controller/class-enform-fields-list-table.php]: Parent function also returns an empty array. I removed it.

**Pull request merged**